docs/guide/writing/beginners-guide-to-docs.rst

```diff
@@ -1,323 +1,210 @@
-Why would we forgo using those tools when writing documentation?
-This workflow is powerful, and familiar to developers.
+First, know which of these common audiences you're writing for:
 
+* Users - want to use your code and don't care how it works.
+* Developers - want to contribute to your code.
 
-Basic example
-~~~~~~~~~~~~~
+Importantly, beware of overusing Frequently Asked Questions (FAQs) as documentation. Although FAQs can be useful starter documentation for new projects until you have time to write better documentation, FAQs quickly outgrow their use due to many drawbacks: 
 
-::
+* Become quickly outdated.
+* Accumulate disparate content on unrelated topics.
+* Become hard to sort and search through as content is added.
+* Are rarely an actual list of frequently asked questions from real users.
+* Tempt you to add content as a quick fix in lieu of creating actual, thorough documentation.
 
-	Resources
-	---------
+Show what problem your code solves
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-	* Online documentation: http://docs.writethedocs.org/
-	* Conference: http://conf.writethedocs.org/
+People use your docs to learn about your project. 
+Perhaps they will hear about your project from others or will find it from a search engine. 
+Regardless, clearly state what your project does and why. 
 
-This will render into a header,
-with a list underneath it.
-The URLs will be hyperlinked automatically.
-It's easy to write,
-still makes sense as plain text,
-and renders nicely into HTML.
+Fabric_ does a great job of this.
 
-README
-~~~~~~
+.. _Fabric: http://docs.fabfile.org/
 
-Your first steps in documentation should go into your README.
-Code hosting services will render your README into HTML automatically if you provide the proper extension.
-It is also the first interaction that most users will have with your project.
-So having a solid README will serve your project well.
+Provide a small code example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Some people even go as far as to `start your project with a README`_
+Show a common example use case for your project. 
 
-.. _start your project with a README: http://tom.preston-werner.com/2010/08/23/readme-driven-development.html
+Requests_ is a great example.
 
-.. _write:
+.. _Requests: https://requests.kennethreitz.org/en/master/
 
-What to write
--------------
+Link to your code and issue tracker
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Now we're getting down to the brass tacks.
-Making sure that you give your users all the information that they need,
-but not too much.
+People sometimes browse the code. They might want to file bugs for issues they've found, 
+so make your code easy to contribute. 
 
-First, you need to ask yourself who you're writing for.
-At first,
-you generally just need to appeal to two audiences:
+The `Python Guide`_ does a good job of this.
 
-* Users
-* Developers
+.. _Python Guide: http://docs.python-guide.org/en/latest/index.html
```

Comment:
I like this framing. Do we want to go as far as pulling the example out and showing it here? Or are we happy just linking?